### DISCUSSION TITLE
exclude some files like those under node_modules

### DISCUSSION THREAD

**Discussion by o-az** [eza-community/eza#763] [Q&A, Howto, Help] [answered]
how to?

**Comment by PThorpe92** [comment, marked as answer]
eza supports git so you can pass the `--git-ignore` flag and seeing that `node_modules` and other directories like it that you would probably want to ignore are almost always in git repositories, this is probably what you are looking for :+1:

**Comment by An0n-01** [comment]
~and other folders (custom) like components/ui for example?~

nevermind. `-I [folder/filename]` did it.
